### DISCUSSION TITLE
Opencore 0.7.5 Update for Intel wifi

### DISCUSSION THREAD

**Discussion by nguyenminhhao9988** [Hasodikis/ThinkPad-460-OpenCore#4] [General]
Hello Hasodikis 
I'm newbie. pls help to update to opencore 0.7.5 Intel Wifi 
thank you

**Comment by k4kanwar** [comment]
Hey bro is this monterey supported?

because the title and the thread names suggest catalina and bigsur at the same time and its confusing
